docs/fleet/machines.md

```diff
@@ -25,63 +25,56 @@ Click the name of a machine to go to that machine's page, where you'll find a va
 
 ## Navigating the machine page
 
-The banner at the top of the machine page displays the machine's location, name, and a dropdown list of all {{< glossary_tooltip term_id="part" text="parts" >}} of that machine.
+Click on the **parts live** display at the top left corner of the page to open a menu with information about each {{< glossary_tooltip term_id="part" text="part" >}} of this machine.
+Once you connect to the `viam-server` instance on a part, this display includes its OS, Host, `viam-server` version, IP addresses, and what time it was last online:
 
-If you've connected your physical machine running `viam-server` to its instance in the Viam app, the banner also displays when the machine was last online, which version of `viam-server` it is running, the host name, the IP address or addresses, and its operating system.
+![The machine page with part menu expanded](/fleet/app-usage/machine-page.png)
 
-![The machine page with menu tabs](/fleet/app-usage/machine-page.png)
+### Set up a new machine
 
-For each machine in your fleet, you start by setting up the machine on the **Setup** tab:
+<!-- TODO R2D2: might need screenshot and needs to be revisited once setup construction is finished -->
 
-### Setup
+To connect to the `viam-server` instance on a part, follow the setup instructions.
+Click **View setup instructions** on the part dropdown menu.
```

Comment:
so is that also where you'd move from one part to another? that feels like it might be confusing?

Reply:
No, i think that's the nested left hand menu where you can move between part cards. Maybe if I make it more clear where this one is?